chain/cosmos/chain_node.go

```diff
@@ -620,7 +620,7 @@ type CodeInfosResponse struct {
 	CodeInfos []CodeInfo `json:"code_infos"`
 }
 
-func (tn *ChainNode) InstantiateContract(ctx context.Context, keyName string, amount ibc.WalletAmount, fileName, initMessage string, needsNoAdminFlag bool) (string, error) {
+func (tn *ChainNode) StoreContract(ctx context.Context, keyName string, fileName string) (string, error) {
```

Comment:
Please add a godoc comment for these public methods, copying from the CosmosChain is fine.

Reply:
Updated with comments